src/Helpers/Session/Session.php

```diff
@@ -0,0 +1,58 @@
+<?php
+namespace Give\Helper\Session;
+
+use function Give\Helpers\Form\Utils\isConfirmingDonation;
+
+/**
+ * Get donor session.
+ *
+ * @return array|string
+ * @since 2.7.0
+ */
+function getSession() {
+	return Give()->session->get( 'give_purchase' );
+}
+
+/**
+ * Store data to donor session.
+ *
+ * @since 2.7.0
+ * @param string $key
+ * @param mixed  $data    Non null value. Key will be unset if value set to null
+ * @param bool   $replace
+ *
+ * @return array|string
+ */
+function storeDataIntoSession( $key, $data, $replace = true ) {
+	$session = getSession();
+
+	if ( null === $data ) {
```

Comment:
Careful that this isn't creating an intended effect, here. I notice that `removeDataFromSession` just calls this and passes null. Is there a scenario where you would actually want to store null in the session? The null value here has an implicit result. Maybe that's fine, you just have to be sure then scenario you're using (i.e. data is null) will never happen and _not_ intend to remove the key. Make sense?

Reply:
@JasonTheAdams nothing will be store in session but instead if pass `null` as data value it will remove that key from session `array`. In `if` condition, I am using `unset` to remove data.